src/pyggdrasil/tree_inference/_config.py

```diff
@@ -0,0 +1,63 @@
+"""Config classes for MCMC sampler,
+ cell simulation and tree inference, distance calculation."""
+
+from pydantic import BaseModel, validator
+
+
+class MoveProbConfig(BaseModel):
+    """Move probabilities for MCMC sampler."""
+
+    prune_and_reattach: float = 0.1
```

Comment:
Glad you started using Pydantic! :slightly_smiling_face: 

It's not necessary to correct this, but in the future you may prefer using constrained floats (see [this](https://docs.pydantic.dev/latest/usage/types/#constrained-types))

Reply:
Great to know - thank you!